.github/workflows/rust.yml

```diff
@@ -109,7 +109,7 @@ jobs:
             # Check WASM examples, which aren't a part of the workspace and so are otherwise missed:
             - name: Cargo check WASM examples
               run: |
-                cargo check --manifest-path examples/wasm-example/Cargo.toml
+                cargo check --manifest-path examples/wasm-example/Cargo.toml --target wasm32-unknown-unknown
```

Comment:
Nit: I think this should already be the case since `target: wasm32-unknown-unknown` means that it's the only target that will be installed, but nothing wrong with having some extra reassurance (I've done similar!)